dist/assembly.xml

```diff
@@ -37,7 +38,7 @@
             <directory>target</directory>
             <outputDirectory/>
             <includes>
-                <include>${server.output.dir.prefix}-core-${project.version}/**</include>
+                <include>${server.output.dir.prefix}-${wildfly.core.release.version}/**</include>
```

Comment:
Note for historical research: this bit is the key change in the PR; before it was using the wrong value, causing the bug.